qpmodel/ExprFunc.cs

```diff
@@ -222,7 +222,6 @@ public override void Bind(BindContext context)
         {
             base.Bind(context);
             type_ = args_()[0].type_;
-            Debug.Assert(type_ is CharType || type_ is VarCharType);
```

Comment:
how about we enhance the assertion that either type_ is char or null?

Reply:
Done